packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -1068,4 +1069,52 @@ export const standardLinterfunctions: FunctionItem[] = [
       return true;
     },
   },
+  {
+    functionName: 'apilintOpenAPIParameterFieldIsDefinedWithinPathTemplate',
+    function: (element: Element) => {
+      if (element.element === 'parameter') {
+        const parameterName = toValue((element as ObjectElement).get('name'));
+        const parameterLocation = toValue((element as ObjectElement).get('in'));
+        const isInPathItemElement =
+          isArrayElement(element.parent) &&
+          includesClasses(['path-item-parameters'], element.parent);
+        const isChildOfOperationElement = element.parent.parent.parent.element === 'operation';
```

Comment:
We can bail out right here, if the parameter is not part of `PathItem.parameters`:

```js
if (!isInPathItemElement) {
  return true;
}
```

Reply:
Next we need to estabilish the `pathItemElement`:

```js
const pathItemElement = parameterElement.parent.parent.parent
```

Reply:
Next we need to estabilish if `Path Item Object` is part of path templating (inside Paths Object) and not part of Webhook, Callback or Components objects:

```js
const isPathItemPartOfPathTemplating = isStringElement(pathItemElement.meta.get('path');
```

Reply:
We can bail out right here, if `Path Item Object` is **not** part of path templating:

```js
if (!isPathItemPartOfPathTemplating) {
  return true
}
```

Reply:
After the above check, we can actually do the lookup of parameter name inside the path template.